p4include/p4d2model_bmss_meta.map

```diff
@@ -0,0 +1,29 @@
+# This file maps p4d2 metadata to the bmv2 simple switch standard and intrinsic metadata
+
+# parser
+parser_input_t.ingress_port       -> standard_metadata.ingress_port
```

Comment:
I think the problem might be that in the original compiler (and specifically in p4-hlir) standard_metadata was instantiated automatically, without the corresponding "header_type" being explicitly declared. Similarly, in simple_switch standard_metadata is instantiated in a very "back-door"-ish way without the type being declared first. 

But going forward, I suggest we should be more formal about these things, otherwise it is too confusing.